web/settings/context.go

```diff
@@ -47,6 +51,34 @@ func onboarded(c echo.Context) error {
 		}
 	}
 	redirect := i.OnboardedRedirection().String()
+
+	// Retreiving client
+	clientID := c.QueryParam("client_id")
+	client, err := oauth.FindClient(i, clientID)
+	if err != nil {
+		return err
```

Comment:
No, on the web onboarding, we have no client, but we should not `return err` here!

Reply:
And this shows the need of unit tests ;-)